docs/Documentation/openvino_ecosystem.md

```diff
@@ -62,7 +62,6 @@ More resources:
 
 **Compile Tool** 
```

Comment:
do you want to keep compile tool and dlwb in ecosystem?

Reply:
Not sure. Same goes for OVTF.

Reply:
> do you want to keep compile tool and dlwb in ecosystem?

No

Reply:
Ok, I'm easily convinced. Let's throw them out.